mmcv/ops/all_pairs_correlation.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from torch import Tensor, nn
+from torch.autograd import Function
+from torch.autograd.function import once_differentiable
+
+from ..utils import ext_loader
+
+ext_module = ext_loader.load_ext(
+    '_ext',
+    ['all_pairs_correlation_forward', 'all_pairs_correlation_backward'])
+
+
+class AllPairsCorrelationFunction(Function):
+
+    @staticmethod
+    def forward(ctx, input1, input2):
+        ctx.save_for_backward(input1, input2)
+        iH1, iW1 = input1.shape[-2:]
+        iH2, iW2 = input2.shape[-2:]
+        batch_size = input1.shape[0]
+        output_size = (batch_size, iH1, iW1, iH2, iW2)
+
+        output = input1.new_empty(output_size)
```

Comment:
suggest using input1.new_zeros to ensure all the elements in output are initialized as zeros.